### ISSUE TITLE
Indentation error showing while starting gulp

### ISSUE THREAD

**Issue by soumyaa1804** [WikiEducationFoundation/WikiEduDashboard#3768]
### What is happening? 
While starting gulp, the execution shows a minor error which could easily be fixed. The error is as follows:
```
  47:10  error  Expected indentation of 10 space characters but found 12  react/jsx-indent

✖ 1 problem (1 error, 0 warnings)
  1 error and 0 warnings potentially fixable with the `--fix` option.
```

### To Reproduce
run `gulp` inside the project repository.

### Expected behavior
Though the error does not hinder the work but this could be improved. 

### Screenshots
![image](https://user-images.githubusercontent.com/40785645/74871225-2089a680-5381-11ea-828b-29f4213f4807.png)

### System Information

 - OS: Linux mint 18.3
 - Browser: chrome
 - Version: 80.0.3987.100

**Comment by ragesoss** [comment]
Hmm... I don't get that error locally.

I wonder what could be causing it. My first idea would be perhaps you have a different version of one of the dependencies. Does it still happen if you run `yarn --ignore-engines` and then start gulp again?

**Comment by soumyaa1804** [comment]
The command gave an error:
```
soumya@Soumya ~/repos/WikiEduDashboard $ yarn --ignore-engines
Usage: yarn [options]

yarn: error: no such option: --ignore-engines
```

**Comment by soumyaa1804** [comment]
When I change the indentation in /WikiEduDashboard/app/assets/javascripts/components/overview/my_articles/components/Categories/List/Assignment/ProgressTracker/ProgressTracker.jsx 47:10
and save it, it loads the gulp again and I don't get the error then.

**Comment by ragesoss** [comment]
Hmm... which version of yarn are you on?

**Comment by soumyaa1804** [comment]
yarn --version
0.22

**Comment by ragesoss** [comment]
That could be the problem. That is a very old version of yarn. If there's not a newer one available in your distro, I think it would help to install one from a different source.

https://classic.yarnpkg.com/en/docs/install/#debian-stable

**Comment by soumyaa1804** [comment]
Yeah! It got fixed! I updated the yarn to 1.22.0 and ran `yarn --ignore-engines` and then ran `gulp` again. Thanks @ragesoss
